package.json

```diff
@@ -34,20 +30,29 @@
   },
   "homepage": "https://github.com/monti-apm/monti-apm-core#readme",
   "devDependencies": {
-    "babel-cli": "6.x.x",
-    "babel-core": "6.x.x",
-    "babel-eslint": "^10.1.0",
-    "babel-polyfill": "6.x.x",
-    "babel-preset-es2015": "6.x.x",
-    "babel-preset-stage-2": "6.x.x",
-    "chai": "^4.2.0",
-    "eslint": "^6.8.0",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.3",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/register": "^7.21.0",
+    "@types/chai": "^4.3.4",
+    "chai": "^4.3.7",
+    "eslint": "^8.37.0",
     "eslint-plugin-babel": "^5.3.0",
     "express": "^4.17.1",
-    "mocha": "^5.2.0"
+    "mocha": "^5.2.0",
+    "nodemon": "^2.0.21",
+    "ws": "^8.13.0"
   },
   "dependencies": {
-    "axios": "^0.21.4",
-    "debug": "^3.2.6"
+    "axios": "^1.3.4",
+    "debug": "^3.2.6",
+    "eventemitter2": "^6.4.9",
+    "faye-websocket": "^0.11.4",
+    "uuid": "^9.0.0"
```

Comment:
Have you checked if these all work in Node 12?

Reply:
I think I did, but will check again.

Reply:
It does

<img width="442" alt="image" src="https://github.com/monti-apm/monti-apm-core/assets/17411182/3e51d63b-e531-47e5-b6fb-bd31b14f0d06">